[OSDOCS#17174]: Make changes as per the errors and warnings for CQA-3 (#114168)
openshift/openshift-docs: security/compliance_operator/co-management/compliance-operator-uninstallation.adoc

toc::[]

[role="_abstract"]

> You can remove the OpenShift Compliance Operator from your cluster by using the {product-title} web console or the CLI.

Reviewer: ```suggestion
You can remove the OpenShift Compliance Operator from your cluster by using the {product-title} web console or the {oc-first}.
```

include::modules/compliance-operator-uninstall.adoc[leveloffset=+1]

include::modules/compliance-operator-cli-uninstall.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

> * xref:compliance-operator-installation.adoc#compliance-operator-installation[Installing the Compliance Operator]

Reviewer: ```suggestion
* xref:../compliance-operator-installation.adoc#compliance-operator-installation[Installing the Compliance Operator]
```

> * xref:compliance-operator-manage.adoc#compliance-operator-understanding[Managing the Compliance Operator]

Reviewer: ```suggestion
* xref:../compliance-operator-manage.adoc#compliance-operator-understanding[Managing the Compliance Operator]
```
I don't know how many directories up these xrefs are, but this format is not correct. See https://docs.google.com/document/d/1sPonVNH_0wvv4Q-Ls00fvJsJsqLRXZkUUjv3xdBOHPg/edit?tab=t.0#heading=h.v2vwbcvofbn9 for examples
